jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/driver/MobileDriverInfos.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.jdi.light.mobile.driver;
+
+import com.epam.jdi.light.driver.get.DriverInfo;
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.ios.IOSDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import java.net.URL;
+
+import static com.epam.jdi.light.driver.get.DriverData.*;
+import static com.epam.jdi.light.driver.get.RemoteDriver.*;
+import static com.epam.jdi.light.mobile.driver.MobileDriverData.*;
+
+/**
+ * Created by Roman Iovlev on 26.09.2019
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+public class MobileDriverInfos {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'MobileDriverInfos' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)